content-elements/form/form-field/prototype/form-field.js

```diff
@@ -46,6 +46,75 @@ Alpine.data('formField', () => ({
       this.requiredErrorElement.style.display = "none";
     }
     this._validateMailInput();
+    this._validateNumberInput();
+    this._validateTextInput();
+    this._validateDateTimeInput();
+  },
+
+  _validateNumberInput() {
+    if (this.inputEl.type === 'number') {
+      let inputValue = parseInt(this.inputEl.value);
+      let minValue = parseInt(this.inputEl.min);
+      let maxValue = parseInt(this.inputEl.max);
+      let valid = minValue <= inputValue && inputValue <= maxValue && inputValue != '';
+
+      if (this.inputEl.value == '' || this.inputEl.value == this.inputEl.defaultValue) {
+        this.invalidErrorElement.setAttribute('hidden', 'true');
+        this.invalidErrorElement.setAttribute('aria-hidden', 'true');
+        return;
+      }
+      if (valid) {
```

Comment:
the if/else clause to set/remove (aria)-hidden should not be duplicated accross the four validation methods. Try to centralize it and use `checkValidity`* to check if the input is valid or not.

* I'm quite sure that `checkValidity` is the right one, but please test it by yourself:  https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/checkValidity